<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface</class>
 <widget class="QWidget" name="interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>139</width>
    <height>55</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Ultracopier</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/systray_Caught_Unix.png</normaloff>:/systray_Caught_Unix.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="text">
       <property name="text">
        <string>Action:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="action">
       <property name="font">
        <font>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Waiting</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="maximum">
        <number>65535</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="toolButton">
       <property name="text">
        <string notr="true">...</string>
       </property>
       <property name="popupMode">
        <enum>QToolButton::InstantPopup</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionAddFile">
   <property name="icon">
    <iconset>
     <normaloff>:/Themes/Clean/resources/add.png</normaloff>:/Themes/Clean/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset>
     <normaloff>:/Themes/Clean/resources/add.png</normaloff>:/Themes/Clean/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
  <action name="actionAddFileToMove">
   <property name="icon">
    <iconset>
     <normaloff>:/Themes/Clean/resources/add.png</normaloff>:/Themes/Clean/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Move file(s)</string>
   </property>
  </action>
  <action name="actionAddFolderToMove">
   <property name="icon">
    <iconset>
     <normaloff>:/Themes/Clean/resources/add.png</normaloff>:/Themes/Clean/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Move folder</string>
   </property>
  </action>
  <action name="actionAddFileToCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/Themes/Clean/resources/add.png</normaloff>:/Themes/Clean/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Copy file(s)</string>
   </property>
  </action>
  <action name="actionAddFolderToCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/Themes/Clean/resources/add.png</normaloff>:/Themes/Clean/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Copy folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
